consul_srv/__init__.py

```diff
@@ -1,21 +1,33 @@
 import requests
 
+from collections import namedtuple
+
 from . import query
 
 __all__ = ["service", "register", "mock", "AGENT_URI"]
 
 AGENT_URI = "127.0.0.1"
 
+TeeConfig = namedtuple('TeeConfig', 'serv_original serv_experimental latency_delta')
+DEFAULT_TEE_SERVICE = 'fore'
+HEADER_SERVICE = 'X-SERVICE-ORIGINAL'
+HEADER_SERVICE_EXP = 'X-SERVICE-EXP'
+HEADER_LAT_DELTA = 'X-SERVICE-LATDELTA-SEC'
```

Comment:
Probably needs to be changed elsewhere, but I'd prefer if this was named something like `X-SERVICE-MAXDELTA-SEC` just to make it clear

Reply:
👍